src/remote.js

```diff
@@ -12,14 +12,27 @@ export async function fetchRemote (name) {
   const protocol = remote.protocol === 'http:' ? 'http:' : 'https:'
   const hostname = remote.hostname || remote.host
   let repo = remote.repo
+  let project = remote.repo
 
   if (/gitlab/.test(hostname) && /\.git$/.test(remote.branch)) {
     // Support gitlab subgroups
     repo = `${remote.repo}/${remote.branch.replace(/\.git$/, '')}`
+    project = `${remote.repo}/${remote.branch.replace(/\.git$/, '')}`
+  }
+
+  if (/dev.azure/.test(hostname)) {
+    repo = `${remote.path}`
+    project = remote.repo
+  }
+
+  if (/visualstudio/.test(hostname)) {
+    repo = `${remote.repo}/${remote.branch}`
+    project = remote.owner
   }
 
   return {
     hostname,
-    url: `${protocol}//${hostname}/${repo}`
+    url: `${protocol}//${hostname}/${repo}`,
+    project: `${protocol}//${hostname}/${project}`
```

Comment:
I think `project` is a weird name for this property. It looks like it's only used as the base url for issues? In which case maybe it should be `issueUrl` or something.

Reply:
Yes, it is used only for the issues url. But, since this url is used by Azure/Visualstudio as a base url for many other project functionalities , I named it 'project'.